docs/src/pages/components/data-grid/filtering/QuickFilteringGrid.tsx

```diff
@@ -78,8 +56,22 @@ function QuickSearchToolbar(props: QuickSearchToolbarProps) {
             </IconButton>
           ),
         }}
+        sx={{
+          width: {
+            xs: 1,
+            sm: 'auto',
+          },
+          m: (theme) => theme.spacing(1, 0.5, 1.5),
```

Comment:
```suggestion
          mt: 1,
          mx: 0.5,
          mb: 1.5,
```
One alternative, not sure if it is worthed.